course-tools-quickstart.md

```diff
@@ -0,0 +1,17 @@
+# Course tools Quickstart
```

Comment:
This assumes that the students work with a studentified (part of the [course management tools](https://github.com/lightbend-training/course-management-tools)) version of the materials. We need to adapt the `run` script on the _RPi's_ to support this. same thing for the `copy` command on the laptop too.
In the short term, it's probably easier to adapt these instructions to work with the master materials.